### ISSUE TITLE
Create a bot for Discord groups

### ISSUE THREAD

**Issue by eshnil** [learn-awesome/learn#193]
Similar to #192

**Comment by TBERB** [comment]
I write discord bots in python. Do you still want this?

**Comment by nileshtrivedi** [comment]
@TBERB Hi Tim, thanks for checking this out. I'd prefer not adding a new programming language to the mix. Webhook handlers would need to make db queries, and if done in Ruby, we can reuse the functionality in all the ActiveRecord models.

**Comment by the-wormhole** [comment]
I have made a few nodejs based discord bots. Can I work on this ?

**Comment by nileshtrivedi** [comment]
@the-wormhole I feel nodejs will be a new addition to the tech stack and will increase complexity.

**Comment by 12345678-nishitagole** [comment]
I would like to work upon this issue , if nobody is assigned.

**Comment by nileshtrivedi** [comment]
@12345678-nishitagole Sure. I've assigned this to you. I'd recommend submitting a plan here before you start writing code.

**Comment by nileshtrivedi** [comment]
No, that would add unnecessary complexity to this project. Take a look at how the existing Slack bot works and use that as a starting point for the Discord bot.

**Comment by Kirito-Excalibur** [comment]
I also wanna help. Discord bot written in ruby? Sure. Assign me

**Comment by GargDivanshu** [comment]
hey i am interested in this issue. You want Discord bot in Ruby? am i right ?

**Comment by bilalshaikh292** [comment]
Hi ! I am interested to contribute . can you assign me the issue ?
